### ISSUE TITLE
node-int64 package not found

### ISSUE THREAD

**Issue by avwie** [LibertyDSNP/parquetjs#148]
Thanks for reporting an issue!

### Steps to reproduce

I am using Yarn 3 and trying to use this library.  However, when the example scripts I get the following error:

`Error: @dsnp/parquetjs tried to access node-int64, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.`

### Any other comments?

I needed to add it in the `yarnrc.yml` as manual resolution:
```
  '@dsnp/parquetjs@*':
    dependencies:
      node-int64: ^0.4.0
```
...

**Comment by wilwade** [comment]
That does look like an oversight. I'll see if I can get it fixed. Thanks for the report!

**Comment by wilwade** [comment]
@avwie Should be fixed in v1.8.5
